docs/configuration.md

```diff
@@ -210,3 +210,21 @@ REMS can be configured to delete applications after a set period of time has pas
 ## Shopping cart
 
 REMS has a shopping cart feature which allows bundling multiple resources into single application. Shopping cart is enabled by default, and it can be enabled or disabled using the `:enable-cart` key in your `config.edn`. See [config-defaults.edn](https://github.com/CSCfi/rems/blob/master/resources/config-defaults.edn) for details.
+
+## Attachment Malware Scanning
+
+REMS can be configured to automatically scan user uploaded files for malware. 
+
+```clojure
+ :malware-scanning {:scanner-path "/path/to/malware/scanner" ; Path to scanner executable
+                    :logging true} ; Whether REMS should log STDERR of malware scanner 
```

Comment:
Replace STDERR with "output"?

Reply:
Possibly `:logging true` should be the default. Otherwise when there is a question, there likely won't be any logs to look at. So something needs to be logged always and perhaps this is just details?

Reply:
I set it to `true`. Though it might be even better to flip the option to `:discard-output false`. As the main reason I included the parameter, is for the situation where this is already logged somewhere else, and there is a desire to not log things twice.

But that might in and of itself be such an esoteric situation (that can be handled simply by piping the outputs somewhere else, such that it doesn't reach REMS), that there is no reason to keep this option.

I would not be opposed to dropping the `:logging` config option altogether.